aws/aws-ecr-event-forwarder/README.md

```diff
@@ -0,0 +1,55 @@
+# AWS Event Forwarder Lambda
+
+This folder contains an [AWS Cloud Formation](https://aws.amazon.com/cloudformation/) template, which sets up necessary resources in AWS to forward AWS Elastic Container Registry (ECR) container vulnerability findings for [basic scanning](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning-basic.html) to the Dynatrace OpenPipeline security event ingest endpoint.
+
+More details can be found in the official documentation:
+
+- [Security events ingest](https://dt-url.net/1d63p0v)
+- [Ingest AWS ECR vulnerability findings](https://dt-url.net/tz03pa8)
```

Comment:
It seems that both of these links don't work (not found)

Reply:
Yes, that is expected. They are seaotter links and once the docs are released, they will work.